### DISCUSSION TITLE
最近是升级了吗， 用的vmess+ws套cf也被墙端口

### DISCUSSION THREAD

**Discussion by greenmickey123** [v2fly/v2ray-core#3210] [General]
最近是升级了吗， 用的vmess+ws套cf也被墙端口

**Comment by panluoyant** [comment]
没有吧

**Reply by greenmickey123** [reply]
我看我的80端口检测一会开启一会关闭的，https://mtool.chinaz.com/port
用外网的端口检测就是一直打开的。。而且我套了cf的cdn理论上不应该呀

**Comment by peaceanddemocracy** [comment]
是的，之前的模式封锁ip影响cdn网站，所以开发出目前的这个端口阻断模式，而且只阻断vpn ws客户端和服务端的端口，其它正常的网站用户客户端不影响，10月已完成测试切换，覆盖大部分常规端口，目前还有个别bug无法及时drop非标端口，预计今年年底前修复。预期效果可以解决大部分因使用vpn ws导致网站cdn ip被误封的情况。

**Reply by greenmickey123** [reply]
有应对的策略吗

**Reply by JuLian1130** [reply]
大佬您说的是墙开发出了端口阻断模式？不用封锁cdn ip了，但vmess+ws套cf更容易被封端口了，是这个意思吗？

**Reply by peaceanddemocracy** [reply]
> 大佬您说的是墙开发出了端口阻断模式？不用封锁cdn ip了，但vmess+ws套cf更容易被封端口了，是这个意思吗？

是的，只阻断端口，3分钟，之后恢复

**Reply by DolorHunter** [reply]
现在推荐用什么配置
